packages/generator/src/generator-without-ts-morph/request-builder/imports.ts

```diff
@@ -1,34 +1,74 @@
 import { unique } from '@sap-cloud-sdk/util';
-import { propertyTypeImportNames } from '../../imports';
-import { externalImportDeclarations, odataImportDeclaration } from '../imports';
-import type { Import } from '@sap-cloud-sdk/generator-common/internal';
+import { StructureKind } from 'ts-morph';
+import type {
+  CreateFileOptions,
+  Import
+} from '@sap-cloud-sdk/generator-common/internal';
 import type { ODataVersion } from '@sap-cloud-sdk/util';
 import type { VdmEntity, VdmProperty } from '../../vdm-types';
+import type { ImportDeclarationStructure } from 'ts-morph';
 
 /**
  * @internal
  */
 export function requestBuilderImportDeclarations(
   entity: VdmEntity,
-  oDataVersion: ODataVersion
-): Import[] {
+  oDataVersion: ODataVersion,
+  options?: CreateFileOptions
+): ImportDeclarationStructure[] {
   return [
-    ...externalImportDeclarations(entity.keys),
-    odataImportDeclaration(
-      [
-        ...requestBuilderImports(entity),
-        'DeserializedType',
-        'RequestBuilder',
-        ...propertyTypeImportNames(entity.keys)
-      ].sort(),
-      oDataVersion
-    ),
-    entityImportDeclaration(entity),
-    ...entityKeyImportDeclaration(entity.keys)
+    {
+      kind: StructureKind.ImportDeclaration,
+      moduleSpecifier: options?.generateESM
+        ? `./${entity.className}.js`
+        : `./${entity.className}`,
+      namedImports: [entity.className]
+    },
+    {
+      kind: StructureKind.ImportDeclaration,
```

Comment:
[q] Why did we change this? I now see an additional import statement `import { RequestBuilder } from '@sap-cloud-sdk/odata-v4'`